scripts/Deploy.s.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Script, stdJson} from "forge-std/Script.sol";
+import {L1ERC20Bridge} from "src/L1ERC20Bridge.sol";
+import {L2ERC20} from "src/L2ERC20.sol";
+import {Constants} from "test/Constants.sol";
+
+/// @notice Deploy L1 bridge and corresponding token to be minted on L2
+contract Deploy is Script, Constants {
+  using stdJson for string;
+
+  function run() public {
+    // Get the contract address for the L1 ERC20Votes token
+    string memory file = "broadcast/DeployFakeERC20.s.sol/43113/run-latest.json";
+    string memory json = vm.readFile(file);
+    address deployedL1Token = json.readAddress(".transactions[0].contractAddress");
+
+    setFallbackToDefaultRpcUrls(false);
+
+    vm.createSelectFork(getChain("polygon_mumbai").rpcUrl);
+
+    // Create L2 ERC20Votes token
+    vm.broadcast();
+    L2ERC20 l2Token = new L2ERC20("Scopeapotomus", "SCOPE", wormholeCoreMumbai);
```

Comment:
We could read the token data from the L1 deployment then use it here, right? Might be a little cleaner. I do love Scopeapotomus though 😂

Reply:
Sorry, I was not clear here. What I actually meant was—knowing the address of the token on the L1—we could start an L1 fork and read the state from the network before switching to the L2 network where we'll do the actual deploy. This can wait until we cleanup the deploy scripts, though.